infrastructure/templates/datafactory/components/template.json

```diff
@@ -2086,143 +2063,6 @@
-                        "policy": {
-                            "timeout": "7.00:00:00",
-                            "retry": 0,
-                            "retryIntervalInSeconds": 30,
-                            "secureOutput": false,
-                            "secureInput": false
-                        },
-                        "userProperties": [],
-                        "typeProperties": {
-                            "source": {
-                                "type": "AzureSqlSource",
-                                "sqlReaderQuery": {
-                                    "value": "SELECT SubjectId FROM dbo.ReleaseSubject RS WHERE RS.ReleaseId = CONVERT(UNIQUEIDENTIFIER, '@{pipeline().parameters.releaseId}')\nAND RS.SubjectId IN (SELECT SubjectId FROM dbo.ReleaseSubject WHERE SubjectId = RS.SubjectId AND ReleaseId != CONVERT(UNIQUEIDENTIFIER, '@{pipeline().parameters.releaseId}'))\n",
-                                    "type": "Expression"
-                                },
-                                "queryTimeout": "02:00:00"
-                            },
-                            "dataset": {
-                                "referenceName": "ds_sql_statistics",
-                                "type": "DatasetReference",
-                                "parameters": {
-                                    "cw_table": "ReleaseSubject"
-                                }
-                            },
-                            "firstRowOnly": false
-                        }
-                    },
-                    {
-                        "name": "Find_Linked_Footnotes",
-                        "type": "Lookup",
-                        "dependsOn": [
-                            {
-                                "activity": "ForEachLinkedSubject",
-                                "dependencyConditions": [
-                                    "Succeeded"
-                                ]
-                            }
-                        ],
-                        "policy": {
-                            "timeout": "7.00:00:00",
-                            "retry": 0,
-                            "retryIntervalInSeconds": 30,
-                            "secureOutput": false,
-                            "secureInput": false
-                        },
-                        "userProperties": [],
-                        "typeProperties": {
-                            "source": {
-                                "type": "AzureSqlSource",
-                                "sqlReaderQuery": {
-                                    "value": "SELECT FootnoteId FROM dbo.ReleaseFootnote RF WHERE RF.ReleaseId = CONVERT(UNIQUEIDENTIFIER, '@{pipeline().parameters.releaseId}')\nAND RF.FootnoteId IN (SELECT FootnoteId FROM dbo.ReleaseFootnote WHERE FootnoteId = RF.FootnoteId AND ReleaseId != CONVERT(UNIQUEIDENTIFIER, '@{pipeline().parameters.releaseId}'))\n",
```

Comment:
I think it would be helpful in the PR description to explain what the purpose of these two 'linked' vars was (as they select from the `private-stats` db) and what they meant in terms of what was/wasn't being deleted by the old version from the `public-stats` db.

It will then be easier to review what the new version is doing to make sure we aren't missing something from before.

Reply:
I've added further description to this PR. I am just guessing as to why these arguments were added - it's more than possible I'm missing their true reason for being added.